src/docs/metis.mdx

```diff
@@ -0,0 +1,163 @@
+    | <Copy value="0x081D1101855bD523bA69A9794e0217F0DB6323ff" label="0x081...ff" /> | <Reference label="L1CrossDomainMessenger" url="https://github.com/MetisProtocol/mvm/blob/develop/packages/contracts/contracts/L1/messaging/L1CrossDomainMessenger.sol" /> | The L1 Cross Domain Messenger contract sends messages from L1 to L2, and relays messages from L2 onto L1. |
+    | <Copy value="0x4200000000000000000000000000000000000007" label="0x420...07" /> | <Reference label="L2CrossDomainMessenger" url="https://github.com/MetisProtocol/mvm/blob/develop/packages/contracts/contracts/L2/messaging/L2CrossDomainMessenger.sol" /> | The L2 Cross Domain Messenger contract sends messages from L2 to L1, and relays messages from L1 onto L2. |
+    
+</Section>
+
+<Section title="RPC-API">
+
+    <Legend />
+
+    The following endpoints behave differently compared to the canonical Ethereum L1 implementation of the JSON RPC API.
+
+    | Method | Params | Rollup Behaviour | Ethereum L1 Behaviour |
+    | :----- | :----- | :--------------- | :-------------------- |
+    | `eth_getBlockByNumber` | Integer block number or string one of `safe`, `latest` or `pending` | Returns information for a given block.<br /><br />Adds additional information for transactions such as l1 block number, l1 timestamp, queue origin and raw transaction. | Returns information for a given block. <Modified /> |
+    | `eth_getBlockByHash` | hash | Returns information for a given block.<br /><br />Adds additional information for transactions such as l1 block number, l1 timestamp, queue origin and raw transaction. | Returns information for a given block. <Modified /> |
+    | `eth_getTransactionByHash` | hash | Returns information for a given transaction. <br /><br />Adds additional information such as l1 block number, l1 timestamp, queue origin and raw transaction. | Returns information for a given transaction. <Modified /> |
+    | `eth_getTransactionReceipt` | hash | Returns information for a given transaction. <br /><br />Adds additional information related to L1 calldata fees. | Returns information for a given transaction. <Modified /> |
+
+</Section>
+
+<Section title="Transaction Fees">
+
+    The L2 fees of Metis work identically to gas fees on Ethereum. Metis transactions must also, however, pay an L1 fee component to cover the cost of their calldata paid in METIS. You should use `eth_gasPrice` to determine the appropriate gas price needed.
+
+</Section>
+
+<Section title="Tools">
+
+    Existing EVM-based tooling is supported such as ethers, web3.js, ethers-rs, hardhat, foundry and truffle.
```

Comment:
Do they have an npm library for helping out with calldata fees or l2 <-> l1 messages?

Reply:
Nothing noteworthy i would say. they have polis: https://github.com/MetisProtocol/polis-sdk, however it seems like an ethers.js ripoff + oauth flow. Should i include it anyways?